nativescript-charter.md

```diff
@@ -0,0 +1,106 @@
+# NativeScript Charter
+
+## Section 0. Guiding Principle
+
+NativeScript is part of the OpenJS Foundation. The project operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1. Scope
+
+NativeScript brings native platform APIs directly into the JavaScript runtime, enriched with robust TypeScript typings, ensuring a seamless and powerful development experience. As an open-source framework for building applications across iOS, visionOS, Android, macOS, and more, it uniquely blends familiar web development paradigms—such as CSS styling and template-driven views—with popular native languages including Swift, Kotlin, Objective-C, and Java. This integration offers developers unparalleled flexibility and ease.
+Its intuitive CLI further simplifies interaction with native platform development processes, making native development approachable for JavaScript developers.
+
+### 1.1: In-scope
+
+Section Intentionally Left Blank
```

Comment:
I believe these sections are mandatory, no?

Reply:
You can leave as is. That's fine.